data/categories_cuisines.txt

```diff
@@ -256,6 +264,7 @@ nl:Diner
 fi:Lounaskahvila
 fr:Diner américain
 de:Diner
+hi:भोजनगृह
```

Comment:
It should be just भोजन , not भोजनगृह which would mean a place for dinner. Not the "cuisine".

```suggestion
hi:भोजन
```